### PR TITLE
feat: viewer blocks api

### PR DESCRIPTION
**Pull request by 0xff-dev**
feat: viewer blocks api
https://github.com/bestchains/bc-explorer/issues/3

### CODE REVIEW

File: pkg/viewer/block.go [hunk 1]
```diff
@@ -5,12 +5,107 @@ Licensed under the Apache License, Version 2.0 (the "License");
...
+	"fmt"
+
+	"github.com/bjwswang/bc-explorer/pkg/models"
+	"github.com/go-pg/pg/v10"
+	"k8s.io/klog/v2"
+)
+
+type BlockArg struct {
+	From, Size         int
+	ChannelName        string
```
Comment (bjwswang) [inline pkg/viewer/block.go:29]: Use `NetworkName` or `Network`

Comment (bjwswang) [inline pkg/viewer/block.go:29]: And this filed is must required

File: pkg/viewer/viewer.go [hunk 2]
```diff
@@ -5,18 +5,89 @@ Licensed under the Apache License, Version 2.0 (the "License");
...
+	arg := BlockArg{
+		From:        ctx.QueryInt("from", 0),
+		Size:        ctx.QueryInt("size", 10),
+		ChannelName: ctx.Query("channel"),
+		StartTime:   int64(ctx.QueryInt("startTime", 0)),
+		EndTime:     int64(ctx.QueryInt("endTime", 0)),
+		BlockNumber: uint64(ctx.QueryInt("blockNumber", 0)),
+		BlockHash:   ctx.Query("blockHash"),
+	}
+	klog.V(5).Infof(" with ctx  %+v arg: %+v\n", *ctx, arg)
```
Comment (bjwswang) [inline pkg/viewer/viewer.go:49]: please add this level to listener

File: pkg/viewer/viewer.go [hunk 3]
```diff
@@ -5,18 +5,89 @@ Licensed under the Apache License, Version 2.0 (the "License");
...
+		bs, _ := json.Marshal(map[string]string{"msg": err.Error()})
+		return ctx.Send(bs)
+	}
+
+	data := map[string]interface{}{
+		"data":  result,
+		"count": count,
+	}
+	bs, _ := json.Marshal(data)
+	return ctx.Send(bs)
```
Comment (bjwswang) [inline pkg/viewer/viewer.go:64]: use `ctx.Json()`

File: pkg/viewer/viewer.go [hunk 4]
```diff
@@ -5,18 +5,89 @@ Licensed under the Apache License, Version 2.0 (the "License");
...
+		BlockHash:   ctx.Query("blockHash"),
+	}
+	klog.V(5).Infof(" with ctx  %+v arg: %+v\n", *ctx, arg)
+	result, count, err := h.block.List(arg)
+
+	ctx.Set("Content-Type", "application/json")
+	if err != nil {
+		ctx.Status(http.StatusInternalServerError)
+		bs, _ := json.Marshal(map[string]string{"msg": err.Error()})
+		return ctx.Send(bs)
```
Comment (bjwswang) [inline pkg/viewer/viewer.go:56]: Is this good enough ? 
https://github.com/bestchains/bc-explorer/blob/main/pkg/listener/handler.go#L61

**Comment by 0xff-dev** [comment]
Update module name. `github.com/bjwswang/bc-explorer` -> `github.com/bestchains/bc-explorer`

**Pull request merged**